portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase

```diff
@@ -588,72 +588,80 @@ definition {
+			Vocabulary.addCP(vocabularyName = "Vocabulary Name 2");
+		}
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Content",
-			panel = "Applications",
-			portlet = "Content Dashboard");
+		task ("Check if vocabularies are avalible") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
 
-		Click(locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_BUTTON");
+			Click(locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_BUTTON");
 
-		SelectFrame(locator1 = "IFrame#MODAL_BODY");
+			SelectFrame(locator1 = "IFrame#MODAL_BODY");
 
-		AssertElementPresent(
-			columnName = "Available",
-			locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_VOCABULARY_NAME",
-			vocabularyName = "Vocabulary Name 1");
+			AssertElementPresent(
+				columnName = "Available",
+				locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_VOCABULARY_NAME",
+				vocabularyName = "Vocabulary Name 1");
 
-		AssertElementPresent(
-			columnName = "Available",
-			locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_VOCABULARY_NAME",
-			vocabularyName = "Vocabulary Name 2");
+			AssertElementPresent(
+				columnName = "Available",
+				locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_VOCABULARY_NAME",
+				vocabularyName = "Vocabulary Name 2");
+		}
 	}
 
 	@description = "LPS-133797 When clicks on the left or right arrows within Configuration and selects one or more vocabularies, the vocabularies go from Available to In use and vice versa"
 	@priority = "3"
 	test MoveVocabulariesAcrossConfiguration {
-		ApplicationsMenu.gotoPortlet(
-			category = "Content",
-			panel = "Applications",
-			portlet = "Content Dashboard");
+		task ("Go to Content Dashboard > Audic Graph configuration") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
 
-		Click(locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_BUTTON");
+			Click(locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_BUTTON");
 
-		SelectFrame(locator1 = "IFrame#MODAL_BODY");
-
-		Click(
-			columnName = "In Use",
-			locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_VOCABULARY_NAME",
-			vocabularyName = "Audience");
+			SelectFrame(locator1 = "IFrame#MODAL_BODY");
+		}
 
-		Click(locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_LEFT_ARROW_BUTTON");
+		task ("Move vocabulary from in use to available") {
+			Click(
+				columnName = "In Use",
+				locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_VOCABULARY_NAME",
+				vocabularyName = "Audience");
 
-		AssertClick(
-			locator1 = "Button#SAVE",
-			value1 = "Save");
+			Click(locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_LEFT_ARROW_BUTTON");
 
-		Click(
-			columnName = "Available",
-			locator1 = "ContentDashboard#AUDIT_GRAPH_CONFIGURATION_VOCABULARY_NAME",
-			vocabularyName = "Topic");
+			AssertClick(
+				locator1 = "Button#SAVE",
+				value1 = "Save");
+		}
 
```

Comment:
The move vocabulary from in use to available should be a macro, having clicks in the .testcase file is not very useful